appium/tests/specs/inbox/CheckReplyAndForwardForEncryptedEmail.spec.ts

```diff
@@ -33,6 +34,6 @@ describe('INBOX: ', () => {
     await NewMessageScreen.clickBackButton();
     await EmailScreen.clickMenuButton();
     await EmailScreen.clickForwardButton();
-    await NewMessageScreen.checkFilledComposeEmailInfo("", forwardSubject, quoteText);
+    await NewMessageScreen.checkFilledComposeEmailInfo("", forwardSubject, quoteText, attachmentName);
```

Comment:
You could also test that the attachment disappears when you tap `deleteAttachmentButton`. It's out of scope of this PR but I don't think we're testing it anywhere.